electron-app/main/menu.ts

```diff
@@ -85,4 +86,99 @@ if (process.platform === 'darwin') {
   MENU.push(HELP_MENU);
 }
 
-export default MENU;
+export function popupContextMenu(
+  window: BrowserWindow,
+  isDevelopment: boolean,
+  props: Electron.ContextMenuParams
+) {
+  const editFlags: any = props.editFlags;
+  const hasText = props.selectionText.trim().length > 0;
+  const roleEnabled = (role: string) => editFlags[`can${capitalize(role)}`] && hasText;
+
+  // Not using the `enable` nor `visible` properties of MenuConstructorOptions
+  // As they do not appear to work w/ Electron 1.8.4 on Mac OS X
+  let ctxMenuTmpl: MenuItemConstructorOptions[] = [];
+
+  if (roleEnabled('cut') && props.isEditable) {
+    ctxMenuTmpl.push({
+      label: 'Cut',
+      accelerator: 'CmdOrCtrl+X',
+      role: 'cut'
+    });
+  }
+
+  if (roleEnabled('copy') && hasText) {
+    ctxMenuTmpl.push({
+      label: 'Copy',
```

Comment:
If you remove this, it'll use the OS's language settings to make a label based on `role`